src/parser/jobs/vpr/modules/snilk.tsx

```diff
@@ -0,0 +1,113 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import Color from 'color'
+import {DataLink} from 'components/ui/DbLink'
+import {JOBS} from 'data/JOBS'
+import {Event, Events} from 'event'
+import {EventHook} from 'parser/core/Dispatcher'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {CounterGauge, Gauge as CoreGauge} from 'parser/core/modules/Gauge'
+import Suggestions, {SEVERITY, Suggestion, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+import DISPLAY_ORDER from './DISPLAY_ORDER'
+
+type GaugeModifier = Partial<Record<Event['type'], number>>
+
+const FADE = 0.25
+const SNILKMODECOST = 50
+const SnilkColor = Color(JOBS.VIPER.colour).fade(FADE)
+
+const SUGGESTION_TIERS = {
+	25: SEVERITY.MEDIUM,
+	50: SEVERITY.MAJOR,
+}
+
+export class snilk extends CoreGauge {
+	static override title = t('vpr.gauge.title')`Serpent's Offerings Gauge`
```

Comment:
I think in the english tooltips it's referred to as "Serpent Offerings" rather than "Serpent's Offerings".